js/store.js

```diff
@@ -0,0 +1,77 @@
+const store = {
+  init() {
+    this.addListeners();
+  },
+
+  getAll() {
+    return this.get();
+  },
+
+  getFirstParties() {
+    /*
+    * similar to getAll() without the thirPartyRequests
+    */
+  },
+
+  async getFirstParty(hostname) {
+    if (!hostname) {
+      return null;
+    }
+
+    const storage = await this.get();
+    return storage.websites[`${hostname}`];
+  },
+
+  async getThirdParties(hostname) {
+    if (!hostname) {
+      return null;
+    }
+
+    const storage = await this.get();
+
+    return storage.websites[`${hostname}`] ?
```

Comment:
Use `storage.websites[hostname]` instead.
Multiline ternary statements like this are actually hard to read, I would personally go for something like:
```
const hostNameData = storage.websites[hostname];
if ("thirdPartyRequests" in hostNameData) {
  return hostNameData.thirdPartyRequests;
}
return null;
```